apps/ledger-live-mobile/src/screens/NotificationCenter/Status.tsx

```diff
@@ -1,102 +1,119 @@
 import React, { useCallback } from "react";
-import { Linking, FlatList } from "react-native";
+import { Linking } from "react-native";
 import { Trans, useTranslation } from "react-i18next";
 import { useFilteredServiceStatus } from "@ledgerhq/live-common/notifications/ServiceStatusProvider/index";
-import type { Incident } from "@ledgerhq/live-common/notifications/ServiceStatusProvider/types";
-import { Alert, Flex, IconBox, Notification, Text } from "@ledgerhq/native-ui";
-import { CheckAloneMedium } from "@ledgerhq/native-ui/assets/icons";
+import { BottomDrawer, Button, Flex, Icons, Text } from "@ledgerhq/native-ui";
+
 import styled, { useTheme } from "styled-components/native";
+
 import { urls } from "../../config/urls";
+import { track, TrackScreen } from "../../analytics";
 
 type Props = {
-  item: Incident;
-  index: number;
+  isOpened: boolean;
+  onClose: () => void;
 };
 
-const MainContainer = styled.SafeAreaView`
-  background-color: ${p => p.theme.colors.background.main};
-  padding: ${p => p.theme.space[6]}px;
-  flex: 1;
-`;
-
-const BorderedIncidentContainer = styled.View`
-  border: 1px solid ${p => p.theme.colors.neutral.c40};
-  padding: ${p => p.theme.space[5]}px;
-  margin-bottom: ${p => p.theme.space[5]}px;
-
-  border-radius: 4px;
-`;
+const DRAWER = "Notification Center Status";
```

Comment:
Maybe put in the variable name that this is related to tracking ? I was not understanding what it was used for before seeing it's use